### PR TITLE
14 user dashboard and registration styling.

### PR DESCRIPTION
**Pull request by muginga72**
1. This PR will close the following tickets: #5 and #14 
2. Was also implemented the user's avatar
3. run _rails db:migrate_ to update the user's table (avatar and username)
4. Run bundle install to update the gem file.

### CODE REVIEW

**Review by Ronapj1991 — APPROVED** [review]
@

File: spec/requests/users_spec.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe "Users", type: :request do
+  describe "GET /index" do
+    pending "add some examples (or delete) #{__FILE__}"
```
Comment (dsinn) [inline spec/requests/users_spec.rb]: Since this is a new feature, please add some automated tests!

Comment (muginga72) [reply spec/requests/users_spec.rb]: Yes

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+class UsersController < ApplicationController
+  before_action :authenticate_user!
+  
+  def dashboard
+    # Your custom logic for the user dashboard
+    @user = current_user
+    puts "User: #{@user.inspect}" # Check if @user is populated
```
Comment (dsinn) [inline app/controllers/users_controller.rb]: Is this for debugging?

File: spec/requests/users_spec.rb [hunk 3]
```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+RSpec.describe "/users", type: :request do
+  
+  # This should return the minimal set of attributes required to create a valid
+  # User. As you add validations to User, be sure to
+  # adjust the attributes here as well.
+  let(:valid_attributes) {
+    skip("Add a hash of attributes valid for your model")
```
Comment (dsinn) [inline spec/requests/users_spec.rb]: It doesn't look like any of the tests have been filled out in this file, other than `GET /new`. They are quite important to keep bugs out of the controller.

**Review by AigulY — APPROVED** [review]

**Comment by dsinn** [comment]
Superseded by:
* #39
* #41

**Pull request closed without merging**